internal/fleet/integration_policy/testdata/TestAccResourceIntegrationPolicySecretsFromSDK/current/integration_policy.tf

```diff
@@ -0,0 +1,115 @@
+variable "policy_name" {
+  description = "The integration policy name"
+  type        = string
+}
+
+variable "secret_key" {
+  description = "The secret key for access"
+  type        = string
+}
+
+variable "integration_name" {
+  description = "The integration name"
+  type        = string
+  default     = "aws_logs"
+}
+
+variable "integration_version" {
+  description = "The integration version"
+  type        = string
+  default     = "1.4.0"
+}
+
+variable "default_region" {
+  description = "AWS default region"
+  type        = string
+  default     = "us-east-1"
+}
+
+provider "elasticstack" {
+  elasticsearch {}
+  kibana {}
+}
+
+resource "elasticstack_fleet_integration" "test_policy" {
+  name    = var.integration_name
+  version = var.integration_version
+  force   = true
+}
+
+resource "elasticstack_fleet_agent_policy" "test_policy" {
+  name            = "${var.policy_name} Agent Policy"
+  namespace       = "default"
+  description     = "IntegrationPolicyTest Agent Policy"
+  monitor_logs    = true
+  monitor_metrics = true
+  skip_destroy    = false
+}
+
+data "elasticstack_fleet_enrollment_tokens" "test_policy" {
+  policy_id = elasticstack_fleet_agent_policy.test_policy.policy_id
+}
+
```

Comment:
I guess it can be removed:

```suggestion
```

Reply:
https://github.com/elastic/terraform-provider-elasticstack/pull/1482/commits/60ba6b309ff098eabb633b9af89ba029a1951fc6